Move to hashed passwords for the web interface (#10157)
PowerDNS/pdns: pdns/dnsdistdist/docs/reference/config.rst

  .. versionchanged:: 1.6.0
    ``statsRequireAuthentication``, ``maxConcurrentConnections`` optional parameters added.

  .. versionchanged:: 1.7.0

>     The ``password``, optional parameter has been deprecated, ``hashedPassword`` added instead.

Reviewer: ```suggestion
    The optional ``password``parameter has been deprecated and replaced with ``hashedPassword``.
```
